src/components/TextInput/BaseTextInput.js

```diff
@@ -263,6 +264,8 @@ class BaseTextInput extends Component {
                                         }}
                                         // eslint-disable-next-line
                                         {...inputProps}
+                                        // eslint-disable-next-line react/jsx-props-no-spreading
+                                        {...disableAutoCorrectIfSecureText}
```

Comment:
What about? 

```suggestion
                                        autoCorrect={this.props.secureTextEntry ? false: undefined}
```

Reply:
This suggestion should be
```suggestion
                                        autoCorrect={this.props.secureTextEntry ? false: this.props.autoCorrect}
```

Don't forget to add the prop definition

Reply:
alright, and `defaultProps` should be always true right?

Reply:
added default false props let me know if you fell it should be by default true changes added

Reply:
I think it is true by default

Reply:
alright added default true